lib/install/management/store_files.go

```diff
@@ -51,18 +51,17 @@ func (d *Dispatcher) DeleteStores(vchVM *vm.VirtualMachine, conf *config.Virtual
 	}
 
 	var errs []string
-	var emptyImages bool
 	var emptyVolumes bool
 	log.Infof("Removing images")
-	if emptyImages, err = d.deleteImages(ds, p); err != nil {
-		errs = append(errs, err.Error())
+	if err = d.deleteImages(conf); err != nil {
+		return err
 	}
 	emptyVolumes, err = d.deleteDatastoreFiles(ds, path.Join(p, volumeRoot), d.force)
 
-	if emptyImages && emptyVolumes {
+	if emptyVolumes {
 		// if not empty, don't try to delete parent directory here
 		log.Debugf("Removing stores directory")
-		if err = d.deleteParent(ds, p); err != nil {
+		if _, err = d.deleteParent(ds, p); err != nil {
```

Comment:
At this point `p` should just be the root of the VCH directory, in your example `VIC` but if a user passes in a custom directory with `--image-store` it will be that value and it gets deleted if it's empty.